.github/workflows/rubocop.yml

```
@@ -7,21 +7,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Ruby 2.4
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: 2.4
-    - name: Cache gems
-      uses: actions/cache@v1
-      with:
-        path: vendor/bundle
-        key: ${{ runner.os }}-rubocop-${{ hashFiles('**/Gemfile.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-rubocop-
-    - name: Install gems
-      run: |
-        bundle config path vendor/bundle
-        bundle install --jobs 4 --retry 3
+        bundler-cache: true
```

```suggestion
        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```

Copied directly from the setup-ruby documentation, otherwise I'll wonder what step ate the explicit `bundle install` that used to be here!

---

... noting also that the explicit `bundle install --without development` skips a couple of gems that aren't needed for unit tests.